src/admin/views.py

```diff
@@ -15,3 +17,6 @@ def get(self, request):
             issues += challenge.self_check()
 
         return FormattedResponse(issues)
+
+def mail_list(request):
```

Comment:
does this have permissions on it?

Reply:
No, and it can't because of how we presently do authentication.

Reply:
I'm somewhat unhappy about this view existing (even if it will be blank when in release) without authentication. An API endpoint to retrieve sent emails could be helpful, however. Consider for a future addition?

Reply:
An API endpoint could work, although the point of this view was to present the sent emails in a format that's easy for a human to work with. If we have a need for programmatic access to that data, that's where that'd make sense.

The lack of authentication is an unfortunate byproduct of how we do auth, that can't easily be added without a major refactor. This view also isn't loaded unless this development email backend is in use.

https://github.com/ractf/core/blob/2746f754eb049429d5a09a2b00f8a056caafbc93/src/admin/urls.py#L13-L14